### DISCUSSION TITLE
License question

### DISCUSSION THREAD

**Discussion by lucboudreau** [LibrePDF/OpenPDF#859] [General]
Hey there. We are reviewing this project for potential usage and had a couple questions about its licensing. The license file included here names two licenses, but it's not clear if this applies to different modules, or if the **whole** project is dual licensed. Can someone form the team confirm how the licenses are applicable please? 

~~Additionally, while browsing the code, we noticed there are snippets taken form other projects licensed under less known licenses, but those aren't listed in OpenPDF. ie. [This class here](https://github.com/LibrePDF/OpenPDF/blob/master/pdf-toolbox/src/main/java/org/librepdf/openpdf/examples/fonts/languages/Chinese.java#L23) which is [licensed under the SIL open font license](https://github.com/adobe-fonts/source-han-serif/blob/release/LICENSE.txt).~~ We have also found Apache 2.0 code sprinkled here and there. Is there an exhaustive list of the code this project uses and the associated licenses?

Thanks for your help!

Updated: I read the SIL snippet a bit fast and now realize it's not the code that's SIL but the required font you download to add a watermark with the tool. Sorry.

**Comment by andreasrosdalw** [comment]
OpenPDF has a dual LGPL and MPL open source license.

https://www.gnu.org/licenses/old-licenses/lgpl-2.1

http://www.mozilla.org/MPL/2.0/
